### PR TITLE
Ruins UT should not randomly allow broken ruins to pass the test anymore

### PR DESCRIPTION
**Pull request by FluffyGhoster**
Ruins UT should not randomly allow broken ruins to pass the test anymore.
Added various exoplanet turfs equivalents.
Changed various turfs on ruins to be exoplanet equivalents and avoid active edges.
Improved ZAS active edge test logging capability, for slightly easier debugging.

### CODE REVIEW

File: code/modules/overmap/exoplanets/decor/turfs/wood.dm [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/turf/simulated/floor/exoplanet/wood
```
Comment (DreamySkrell) [inline code/modules/overmap/exoplanets/decor/turfs/wood.dm:1]: I am very much against duplicating all of these turf definitions to have /exoplanet/ and non-/exoplanet/ variants. It is pure ass and awful for like a million reasons.

If the problem is that (for example) konyang ruins fail the zas tests on airless planets, the solution should be to test konyang ruins only on konyang.

**Review by DreamySkrell — CHANGES_REQUESTED** [review]
as commented above

File: code/modules/overmap/exoplanets/decor/turfs/wood.dm (hunk 1 above)
Comment (DreamySkrell) [reply code/modules/overmap/exoplanets/decor/turfs/wood.dm:1]: Alternative would be to remove the /exoplanet/ subtype and just make all turfs act like current exoplanet turfs do (and copy the planet's atmos). But that is not good for other reasons as well.

**Comment by FluffyGhoster** [comment]
!review

**Pull request merged**